lib/wp2middleman/frontmatter.rb

```diff
@@ -10,7 +10,7 @@ def initialize(post, include_fields: [])
     def post_data
       data = {
         'title' => post.title,
-        'date' => post.date_published,
+        'date' => post.date_time_published,
```

Comment:
I see. Am I correct in inferring that `#valid?` should then be made more robust to better protect against errors?